core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/start/ClusterStartFlowConfig.java

```diff
@@ -38,8 +40,12 @@ public class ClusterStartFlowConfig extends StackStatusFinalizerAbstractFlowConf
                     .event(CLUSTER_START_EVENT)
                     .noFailureEvent()
 
-                    .from(CLUSTER_START_UPDATE_PILLAR_CONFIG_STATE).to(UPDATING_DNS_IN_PEM_STATE)
+                    .from(CLUSTER_START_UPDATE_PILLAR_CONFIG_STATE).to(CLUSTER_DB_CERT_REPLACEMENT_STATE)
                     .event(CLUSTER_START_PILLAR_CONFIG_UPDATE_FINISHED_EVENT)
+                    .noFailureEvent()
+
+                    .from(CLUSTER_DB_CERT_REPLACEMENT_STATE).to(UPDATING_DNS_IN_PEM_STATE)
```

Comment:
* I think in this case `CLUSTER_DB_CERT_REPLACEMENT_STATE` shall come before `CLUSTER_START_UPDATE_PILLAR_CONFIG_STATE`. The latter prepares and pushes down the pillars to the cluster & runs Salt high state, and that is how we are deploying the DB root certs.
* CM Server is set to autostart, so it is launched automatically once the VM gets started. Once the DB root certs have been replaced in Salt pillars & redeployed to the cluster, CM Server will have to be restarted in order to pick up the changed cert bundle. This may or may not require a separate flow step, though, so I leave it to you to add it later as necessary. :smile: (The CM Server restart may be also incorporated into the high state.)